apps/src/templates/levelSummary/summary.module.scss

```diff
@@ -98,23 +105,39 @@ label.sectionSelector {
     column-gap: 48px;
     padding-top: 24px;
     padding-bottom: 12px;
+
+    .studentResponseBlock {
+      margin-bottom: 16px;
```

Comment:
Can you confirm that these don't get split between columns (e.g. text split between two columns or name and text on two separate columns) when there's a really long response?

We may need to add `display: inline-block` here

Reply:
Yep - it was actually breaking earlier which caused some moving around of CSS.  I took a video of the long test below to show it working.

https://github.com/user-attachments/assets/8a74709f-7265-46de-897e-2eea2c186c0b